ome-dundeeomero.yml

```diff
@@ -250,15 +253,14 @@
       omero.jvmcfg.system_memory: 17000
       omero.ldap.base: "{{ omero_server_ldap_base }}"
       omero.ldap.config: true
-      omero.ldap.new_user_group: default
       omero.ldap.urls: "{{ omero_server_ldap_urls }}"
       omero.mail.config: true
       omero.mail.from: "{{ omero_server_mail_from }}"
       omero.mail.host: "{{ omero_server_mail_host }}"
-      omero.new_user_group: "My Data"
+      omero.ldap.new_user_group: "My Data"
```

Comment:
This one will need to go along with a `bin/omero obj update ExperimenterGroup ... name="My Data"`

Reply:
@joshmoore - ordinarily, (for anyone following along) yes. But in the case of this particular server, we've renamed the group already. cf (Internal to OME) https://trello.com/c/BHPI0xPs

Reply:
The potential cleanup step is to move users to "My Data" who are in the previous `omero.ldap.new_user_group`. Not sure of implications for data, i.e. if this needs to be a "move the data first, then change the group membership".

Reply:
>  But in the case of this particular server, we've renamed the group already. 

Which for me means, `Done` :+1: